src/stylesheets/settings/_settings-utilities.scss

```diff
@@ -17,7 +17,7 @@ https://designsystem.digital.gov/utilities
 ----------------------------------------
 */
 
-$utilities-use-important: false !default;
+$utilities-use-important: true !default;
```

Comment:
Just double-checking. Was this for testing or will utilities be `!important` from now on?

Reply:
Just for testing, I'll revert